src/components/expression/expression-renderer.js

```diff
@@ -218,17 +218,20 @@ genotet.ExpressionRenderer.ZoomStatus = function(params) {
 };
 
 /** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_HEIGHT_PERCENTAGE = 0.2;
+genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_HEIGHT_PERCENTAGE = 0.35;
 
 /** @const {number} */
 genotet.ExpressionRenderer.prototype.DEFAULT_TFA_PROFILE_HEIGHT_PERCENTAGE =
-  0.2;
+  0.35;
 
 /** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_HEIGHT = 25;
+genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_HEIGHT = 20;
 
 /** @const {number} */
-genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_MARGIN = 10;
+genotet.ExpressionRenderer.prototype.DEFAULT_PROFILE_LEGEND_MARGIN = 5;
+
+/** @const {number} */
```

Comment:
Unclear meaning of "LEGEND_CORNERS". I think you should either say "PROFILE_LEGEND_CORNER_RADIUS". DEFAULT is redundant as you do not have other values applied.